packages/feedback-spa/package.json

```diff
@@ -50,7 +52,7 @@
     "@angular/language-service": "10.1.2",
     "@types/jest": "26.0.14",
     "@types/node": "12.12.62",
-    "apollo-angular": "2.0.4",
+    "apollo-angular": "1.10.0",
```

Comment:
Why this is downgraded?.

Reply:
2 isn't supported by angular 10 yet

Reply:
@hybridx said that it was breaking app during last week's call.

Reply:
<img width="1369" alt="Screenshot 2020-09-21 at 3 14 07 PM" src="https://user-images.githubusercontent.com/12994292/93752990-1a142380-fc1d-11ea-8fbd-2da6aba152fd.png">

Reply:
Fixed with the new config

Reply:
So the version of `apollo-angular` would be `1.10.0` ?

Reply:
No it is now 2.0.4 the latest. For that I had to change a few imports and follow the documentation 

```
import { NgModule } from '@angular/core';
import { HttpClientModule } from '@angular/common/http';
import { APOLLO_OPTIONS } from 'apollo-angular';
import { HttpLink } from 'apollo-angular/http';
import { InMemoryCache, ApolloLink } from '@apollo/client/core';
import { setContext } from '@apollo/client/link/context';
import { environment } from 'src/environments/environment';

const uri = environment.API_URL;

export function provideApollo(httpLink: HttpLink) {
  const basic = setContext((operation, context) => ({
    headers: {
      Accept: 'charset=utf-8'
    }
  }));

  const auth = setContext((operation, context) => ({
    headers: {
      Authorization: window.OpAuthHelper?.jwtToken
      ? 'Bearer ' + window.OpAuthHelper.jwtToken
      : '',
    },
  }));

  const link = ApolloLink.from( [ basic, auth, httpLink.create( { uri } ) ] );
  const cache = new InMemoryCache();

  return {
    link,
    cache
  };
}
@NgModule({
  exports: [
    HttpClientModule,
  ],
  providers: [{
    provide: APOLLO_OPTIONS,
    useFactory: provideApollo,
    deps: [HttpLink]
  }]
})
export class GraphQLModule {}
```

Reply:
@riginoommen Why did you merge this PR? There are clearly 2 instances of apollo-angular being declared with different versions in this file.